### ISSUE TITLE
Unable to access the BrowserType class.

### ISSUE THREAD

**Issue by Xen0byte** [microsoft/playwright-dotnet#1522]
Is this by design? In my framework, I used to do something along the lines of...

```
var browser = (browserType.ToLower()) switch
{
    BrowserType.Chromium => await playwright.Chromium.LaunchAsync(launchOptions),
    BrowserType.Firefox => await playwright.Firefox.LaunchAsync(launchOptions),
    BrowserType.Webkit => await playwright.Webkit.LaunchAsync(launchOptions),
    _ => throw new ArgumentOutOfRangeException(nameof(browserType), browserType,
        $"Invalid Browser Type: {browserType} (Expected: Chromium\\Firefox\\WebKit)")
};
```
...but now BrowserType is internal and the above is no longer possible. I am aware that the easy fix would just be to hard-code in the string values, but I would prefer to not have to do that.

**Comment by pavelfeldman** [comment]
BrowserType was internal, only IBrowserType is accessible. So yes, the hard-coded string is the way to go now, in most cases (test harness) you anyways pass it as a string.

**Comment by avodovnik** [comment]
#1529 is a proposal to introduce a `BrowserTypes` class which exposes the constants for browser names. Just as a note, personally, I dislike using magic strings as well, but, equally, this has a simple "workaround" so we might decide to not increase the API surface.

**Comment by Xen0byte** [comment]
I'm happy with whatever decision you guys will decide to take. Thank you for taking this suggestion into consideration.

**Comment by avodovnik** [comment]
We've merged #1529 and it exposes `BrowserType`. We'll push a preview version shortly, but I'm closing this issue for now.
